cmake/FPrime.cmake

```diff
@@ -141,7 +141,7 @@ macro(fprime_setup_override_targets)
 endmacro(fprime_setup_override_targets)
 
 macro(fprime_initialize_build_system)
-    cmake_minimum_required(VERSION 3.16)
+    cmake_minimum_required(VERSION 3.13)
```

Comment:
Why is this changed?

Reply:
Good catch, thank you! I am unsure why it changed; it shouldn't have. 3.16 is what is in devel right now.